src/index.html

```diff
@@ -11,6 +11,10 @@
       rel="stylesheet"
       href="./styles/main.scss"
     />
+    <link
+    rel="stylesheet"
+    href="./styles/main.css"
+    />
```

Comment:
It is unnecessary to link both a SCSS and a CSS file. SCSS is a preprocessor and needs to be compiled into CSS. Ensure that the SCSS file is compiled to CSS and only the CSS file is linked here.